<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Qnx::Internal::BlackBerryInstallWizardNdkPage</class>
 <widget class="QWidget" name="Qnx::Internal::BlackBerryInstallWizardNdkPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Select Native SDK path:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="ndkPathListWidget"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
